tests/CMakeLists.txt

```diff
@@ -43,12 +43,12 @@ sleigh_add_optional_defines(sleigh_ghidra_test PRIVATE)
 
 add_test(
   NAME sleigh_ghidra_unittest
-  COMMAND sleigh_ghidra_test -sleighpath "${PROJECT_BINARY_DIR}" unittests
+  COMMAND sleigh_ghidra_test -sleighpath "${PROJECT_BINARY_DIR}/specfiles" unittests
```

Comment:
Not sure what's going on and how this was working initially, but I need this change to run locally.

Reply:
Also, use `spec_files_build_dir` instead of recreating the path from `PROJECT_BINARY_DIR`.

Reply:
Ok, I was just confused. This isn't necessary.